src/Tools/Consolonia.Templates/README.md

```
@@ -0,0 +1,20 @@
+
+# Consolonia.Templates
+This is a package which adds Consolonia templates to dotnet. 
+
+# Installation
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)